plugins/scaffolder-backend/src/scaffolder/tasks/TaskWorker.ts

```diff
@@ -166,6 +174,17 @@ export class TaskWorker {
   }
 
   async runOneTask(task: TaskContext) {
+    await this.auditor?.createEvent({
```

Comment:
Should we call `auditorEvent?.success()` when task complete with 'completed' and call `auditorEvent?.fail()` when task complete with 'failed'?